src/promptflow/promptflow/_sdk/_service/static/index.html

```diff
@@ -3,11 +3,12 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="icon" href="{{ url_for('static', filename='favicon.ico') }}" />
-    <title>Promptflow traces</title>
+    <link rel="icon" href="/v1.0/ui/traces/assets/favicon.ico" />
+    <title>Trace View</title>
+    <script type="module" crossorigin src="/v1.0/ui/traces/assets/index-4cTGdARK.js"></script>
```

Comment:
crossorigin?

Reply:
not sure, these files will be maintained by UX devs; any comment on this? maybe I need to engage UX dev on this